From cca8fafb1ffe3ca2d9c9a14afbf3fcda6a0f055c Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Tue, 2 Jul 2013 22:35:05 -0700
Subject: [PATCH 1385/1475] MR1: CLOUDERA-BUILD. Avoid port conflicts in JT HA tests. Changes missed earlier
 (cherry picked from commit 8105fa54be0df560c86fb33282ad1f202ab11ca1)
 (cherry picked from commit 6422e36d807a1a70117a9cb9e92c4c4902db8582)

---
 .../org/apache/hadoop/mapred/TestHAStress.java     |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestHAStress.java b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestHAStress.java
index 3f045d8..4f13985 100644
--- a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestHAStress.java
+++ b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestHAStress.java
@@ -124,6 +124,7 @@ public class TestHAStress {
     conf.setLong(HAUtil.MR_HA_ACTIVE_CHECK_MILLIS, 1000);
     conf.setBoolean("mapred.job.tracker.persist.jobstatus.active", true);
     conf.setInt("mapred.job.tracker.persist.jobstatus.hours", 1);
+    conf.set("mapred.job.tracker.persist.jobstatus.dir", "/tmp/jobtracker/jobsInfo");
     cluster = new MiniMRHACluster(conf);
     cluster.getJobTrackerHaDaemon(0).makeActive();
     cluster.startTaskTracker(0, 1);
-- 
1.7.0.4

